app/src/main/kotlin/cz/covid19cz/erouska/AppConfig.kt

```diff
@@ -122,6 +122,8 @@ object AppConfig {
         get() = firebaseRemoteConfig.getBoolean("v2_efgsTravellerDefault")
     val howItWorksUITitle
         get() = firebaseRemoteConfig.getString("v2_howItWorksUITitle")
+    val howItWorksEvalContent
```

Comment:
When adding new localizable RC key, please follow this guide: https://github.com/covid19cz/erouska-remote-config#how-to-add-new-localizable-rc-key

Reply:
I am following that guide, I am at no.4

Reply:
So go ahead with no. 5 and 6 :)